src/angular/controllers/ctrl-my-widgets-selected.test.js

```diff
@@ -0,0 +1,105 @@
+describe('MyWidgetSelectedController', () => {
```

Comment:
Since the entire My Widgets page and all subsequent files were rewritten in React, this file isn't being used any more. I'll add another commit to remove it.